docs/sources/alerting/unified-alerting/message-templating/_index.md

```diff
@@ -52,12 +52,41 @@ The `define` tag in the Content section assigns the template name. This tag is o
 
 Use caution when deleting a template since Grafana does not prevent you from deleting templates that are in use.
 
+### Nested templates
```

Comment:
`Embedded templates` maybe? Go's template language docs call this ["Nested templates"](https://pkg.go.dev/text/template#hdr-Nested_template_definitions) so that's what I went with here.